docs/conf.py

```diff
@@ -77,6 +77,7 @@
 .. |components_type| replace:: Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[:class:`disnake.ui.WrappedComponent`]]]]
 .. |resource_type| replace:: Union[:class:`bytes`, :class:`.Asset`, :class:`.Emoji`, :class:`.PartialEmoji`, :class:`.StickerItem`, :class:`.Sticker`]
 .. _coroutine_link: https://docs.python.org/3/library/asyncio-task.html#coroutine
+.. |vnext| replace:: 2.8
```

Comment:
is this being used anywhere?

Reply:
no, i'll remove it

Reply:
https://github.com/DisnakeDev/disnake/pull/844/commits/1a636b55559bb7687d452d44192b98060978a52e